src/python/pants/backend/python/register.py

```diff
@@ -69,9 +69,9 @@ def build_file_aliases():
 
 
 def register_goals():
```

Comment:
If any of these have options that people might have been using, they should get deprecated options scopes added.